dist/index.js

```diff
@@ -129,7 +151,7 @@ var HCaptcha = function (_React$Component) {
             src = frames[k].getAttribute("src");
             title = frames[k].getAttribute("title");
 
-            if (src.includes(hCaptchaVars.domain) && title.includes(hCaptchaVars.iframe_title)) foundFrame = nodes[i]; //Compare iframe source and title to find correct iframe appended to body - hCaptcha
+            if (src.includes(hCaptchaVars.domain) && title.includes(hCaptchaVars.iframe_title)) foundFrame = nodes[i]; //Compare iframe source and title to find correct iframe appeneded to body - hCaptcha
```

Comment:
typo

Reply:
interesting, this change isn't by me. Seems the typo is already present in the source by someone else.

Reply:
Really? It seems as though its changing in this PR?

Reply:
@gaieges yeah it's because there are previous commits to the source files but `dist/` is older (it wasn't rebuilt during those commits).

Reply:
Ahhh gotcha!